tests/functional/object/test_object_browser_client_provider_ui.py

```
@@ -342,161 +263,371 @@ def test_object_browser_list_objects_with_folders(self, project_factory):
-            # Verify S3 sign-in success
             assert s3_login2.is_signed_in(), "S3 login failed on client2"
             logger.info("Successfully signed in to object browser on client2")
 
-            # Verify client2 sees its own bucket
             buckets_list = bucket_ui2.get_buckets_list()
             logger.info("Buckets visible on client2: %s", buckets_list)
-            assert (
-                client2_bucket_name in buckets_list
-            ), f"Client2 bucket '{client2_bucket_name}' not found in bucket list"
+            assert c2["bucket_name"] in buckets_list, (
+                "Client2 bucket '%s' not found in bucket list" % c2["bucket_name"]
```

Ruff (0.16.0)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)